OpenRA.Game/Sound/Sound.cs

```diff
@@ -69,7 +69,8 @@ ISoundSource LoadSound(string filename)
 						return soundEngine.AddSoundSourceFromMemory(rawData, channels, sampleBits, sampleRate);
 				}
 
-				throw new InvalidDataException(filename + " is not a valid sound file!");
```

Comment:
The design pattern with all our `*Loader`s is the same - sprites, sound, and soon packages - throw on invalid files (yes, the message should be something like "Could not find a suitable SoundLoader for file X").

Also
>[23:01] <+penev> ]later Mailaender I guess the RA2 wavs just need a slight tweak in the "is this a real wav file" checks